src/main/java/com/atlassian/bitbucket/jenkins/internal/trigger/BitbucketWebhookTriggerImpl.java

```diff
@@ -301,7 +301,7 @@ private void registerWebhook(Item item, BitbucketSCMRepository repository,
         }
 
         @SuppressFBWarnings(value = "NP_NULL_ON_SOME_PATH", justification = "If jenkinsProvider is null we inject one, this is a false positive")
-        private boolean webhookExists(Job<?, ?> project, BitbucketSCM input) {
+        private boolean webhookExists(@Nullable Job<?, ?> project, BitbucketSCM input) {
```

Comment:
I think we should have `CheckForNull` instead, as that is what Jenkins uses for this field.